OrrRock/OrrRock/ViewControllers/OnBoardingViewController/OnBoardingViewController.swift

```diff
@@ -82,66 +38,25 @@ class OnBoardingViewController: UIViewController , UISheetPresentationController
         return btn
     }()
     
+    var currentIndex: Int?
+    var pendingIndex: Int?
+    var index = 0
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        setUpDelegate()
-        setUpLayout()
-        // Do any additional setup after loading the view.
-    }
-    
-    
-    private func setUpDelegate(){
-        self.isModalInPresentation = true
-        sheetPresentationController.delegate = self
-        sheetPresentationController.selectedDetentIdentifier = .large
-        sheetPresentationController.prefersGrabberVisible = false
-        sheetPresentationController.detents = [.large()]
-        
+        setDelegate()
+        setLayout()
     }
+}
+
+extension OnBoardingViewController : UIPageViewControllerDelegate{
     
-    private func setUpLayout(){
-        view.backgroundColor = .orrWhite
-        view.addSubview(titleLabel)
-        titleLabel.snp.makeConstraints {
+    func setLayout(){
```

Comment:
```suggestion
    func setLayout() {
```
p1;

코드 컨벤션 지켜주세요!
`private`가 있어도 좋지 않을까요?